output.R

```diff
@@ -242,24 +147,19 @@ if (comp == "Exit") {
       }
     }
   }
-} else {
-  # Select an output module if not defined by readArgs
-  if (!exists("output")) {
-    output <- choose_module("./scripts/output/single",
-                            "Please choose the output module to be used for output generation")
-  }
-
+} else { # comp = single
   # Select an output directory if not defined by readArgs
   if (!exists("outputdir")) {
-    if (!exists("remind_dir")) {
-      temp <- choose_folder("./output", "Please choose the run(s) to be used for output generation")
-      outputdirs <- temp
-      for (i in seq_along(temp)) {
-        outputdirs[i] <- file.path("output", temp[i])
-      }
-    } else {
-      temp <- choose_folder(remind_dir, "Please choose the runs to be used for output generation")
-      outputdirs <- temp
+    dir_folder <- if (exists("remind_dir")) remind_dir else "./output"
+    dirs <- basename(dirname(Sys.glob(file.path(dir_folder, "*", "fulldata.gdx"))))
+    # DK: The following outcommented lines are specially made for listing results of coupled runs
+    # runs <- findCoupledruns(folder)
+    # dirs <- findIterations(runs,modelpath=folder,latest=TRUE)
+    # dirs <- sub("./output/","",dirs)
+    temp <- chooseFromList(dirs, type = "runs to be used for output generation", returnBoolean = FALSE,
+                                      multiple = TRUE)
+    outputdirs <- file.path("output", temp)
```

Comment:
duplicated code - maybe move to a separate function after all?

Reply:
wasn't even necessary. Was a completely redundant structure:
```
if () {
  thecode
  ...
} else {
  thecode
  ...
}
```
So simply moved it before the if.